Externals/SlippiRustExtensions/jukebox/src/fst.rs

```diff
@@ -0,0 +1,206 @@
+use crate::tracks::{get_track_id_by_filename, TrackId};
```

Comment:
One thing I've done is group imports like:

- `std`
- `local`

Any thoughts on that? Mostly just musing on keeping things scan-able across the workspace. I might be inadvertently doing what [https://rust-lang.github.io/rustfmt/?version=v1.4.38&search=#StdExternalCrate%5C%3A](https://rust-lang.github.io/rustfmt/?version=v1.4.38&search=#StdExternalCrate%5C%3A) is already doing and maybe I should just add that to the fmt rules...